tests/test_bgp_neighbor.rb

```diff
@@ -265,6 +265,26 @@ def test_dynamic_capability
     end
   end
 
+  def test_bfd
+    %w(default test_vrf).each do |vrf|
+      neighbor = create_neighbor(vrf)
+      if platform == :ios_xr
```

Comment:
Will do. The reason I did it this way was because the test_dynamic_capability was written this way before by someone and I just wanted to keep this new property the same way. May be this file is very old and it does not use validate_property_excluded method. 
Anyway, will change it